apps/backend/cypress/e2e/user.cy.ts

```diff
@@ -1,74 +1,253 @@
-import { UserPublicDTO } from '@datatlas/shared/models';
-
 describe('USER ACTIONS', () => {
-  const test_user = {
-    username: 'utilisateur_test13',
-    password: 'utilisateur_test_pw',
-    role: 'editor',
+  /*
+      TEST TO MAKE IN THIS ORDER :
+      - Test reaching API (really useful).
+      - Creating, reading, updating and deleting a new user as editor wth fake jwt.
+      - Creating, reading, updating and deleting a new user as editor with correct jwt.
+      - Creating, reading, updating and deleting a new user as admin with fake jwt.
+      - Creating, reading, updating and deleting a new user as admin with correct jwt
+   */
+  const user_test_editor = {
+    username: 'user_test_editor_20',
+    password: 'user_test_pw',
+    role: 'EDITOR',
     active: true,
   };
-  const modified_test_user = {
-    username: 'modified_utilisateur_test13',
-    password: 'modified_utilisateur_test_pw',
-    role: 'editor',
-    active: false,
+  const user_test_admin = {
+    username: 'user_test_admin_20',
+    password: 'user_test_pw',
+    role: 'ADMIN',
+    active: true,
   };
-  let id_test_user = null;
+  let jwtEditorUser;
+  let idEditorUser;
+  let jwtAdminUser;
+  let idAdminUser;
+  let idUserTestEditor;
```

Comment:
Github action seems to have noticed some unused vars

`'idAdminUser' is assigned a value but never used`

`'idUserTestAdmin' is assigned a value but never used`

Reply:
Done.